debian/usr.bin.snap-confine

```diff
@@ -112,6 +114,11 @@
     mount options=(rw bind) /snap/ubuntu-core/*/libx32/ -> /libx32/,
     mount options=(rw bind) /snap/ubuntu-core/*/lib64/ -> /lib64/,
     mount options=(rw bind) /snap/ubuntu-core/*/usr/ -> /usr/,
+    mount options=(rw bind) /snap/ubuntu-core/*/etc/alternatives/ -> /etc/alternatives/,
+
+    # Allow snaps to share content amongst themselves.
+    mount options=(rw bind) /snap/*/** -> /snap/*/**,
+    mount options=(ro bind) /snap/*/** -> /snap/*/**,
```

Comment:
From IRC discussion (thanks @zyga for remembering the deny rules):
\# but don't share /snap/bin
deny audit mount /snap/bin/** -> /**,
deny audit mount /** -> /snap/bin/**,